src/OpenLoco/GameCommands/RemoveTree.cpp

```diff
@@ -0,0 +1,59 @@
+#include "../Economy/Economy.h"
+#include "../Economy/Expenditures.h"
+#include "../Map/TileManager.h"
+#include "../Objects/ObjectManager.h"
+#include "../Objects/TreeObject.h"
+#include "../OpenLoco.h"
+#include "../S5/S5.h"
+#include "GameCommands.h"
+
+using namespace OpenLoco::Interop;
+
+namespace OpenLoco::GameCommands
+{
+    // 0x004BB392
+    static uint32_t removeTree(Map::Pos3 pos, uint8_t type, uint8_t elementType, uint8_t flags)
```

Comment:
pass by const reference the map pos. Pass the rest of the parameters by const. 
I somewhat wonder if you should just pass in the arguments struct by const reference. It keeps the arguments nicely coupled together.